Fix Gosec errors

We need to have better error handling

```bash
[/home/runner/work/kuberhealthy-aws-iam-role-check/kuberhealthy-aws-iam-role-check/main.go:119] - G104 (CWE-703): Errors unhandled. (Confidence: HIGH, Severity: LOW)
    118: 			log.Infoln("Recovered panic:", r)
  > 119: 			checkclient.ReportFailure([]string{r.(string)})
    120: 		}



[/home/runner/work/kuberhealthy-aws-iam-role-check/kuberhealthy-aws-iam-role-check/main.go:130] - G104 (CWE-703): Errors unhandled. (Confidence: HIGH, Severity: LOW)
    129: 			log.Debugln(err)
  > 130: 			checkclient.ReportFailure([]string{err.Error()})
    131: 			return



[/home/runner/work/kuberhealthy-aws-iam-role-check/kuberhealthy-aws-iam-role-check/main.go:138] - G104 (CWE-703): Errors unhandled. (Confidence: HIGH, Severity: LOW)
    137: 
  > 138: 	checkclient.ReportSuccess()
    139: 



Summary:
   Files: 2
   Lines: 315
   Nosec: 0
  Issues: 5

##[error]Process completed with exit code 1.
```

--- a/main.go
+++ b/main.go
@@ -95,7 +95,10 @@ func main() {
 
 	if arn.IsARN(targetArnEnv) != true {
 		log.Errorln("ASSUMED_ROLE_ARN environment variable is not ARN:", targetArnEnv)
-		checkclient.ReportFailure([]string{"ASSUMED_ROLE_ARN environment variable is not ARN"})
+		err = checkclient.ReportFailure([]string{"ASSUMED_ROLE_ARN environment variable is not ARN"})
+		if err != nil {
+			log.Println(err.Error())
+		}
 		return
 
 	}
@@ -104,7 +107,10 @@ func main() {
 	sess = createAWSSession()
 	if sess == nil {
 		err = fmt.Errorf("nil AWS session: %v", sess)
-		checkclient.ReportFailure([]string{err.Error()})
+		err = checkclient.ReportFailure([]string{err.Error()})
+		if err != nil {
+			log.Println(err.Error())
+		}
 		return
 	}
 
@@ -116,7 +122,10 @@ func main() {
 		r = recover()
 		if r != nil {
 			log.Infoln("Recovered panic:", r)
-			checkclient.ReportFailure([]string{r.(string)})
+			err = checkclient.ReportFailure([]string{r.(string)})
+			if err != nil {
+				log.Println(err.Error())
+			}
 		}
 	}()
 
@@ -127,7 +136,10 @@ func main() {
 			err = fmt.Errorf("Error occurred during runArnCheck: %w", err)
 			log.Infoln("IAM check failed")
 			log.Debugln(err)
-			checkclient.ReportFailure([]string{err.Error()})
+			err = checkclient.ReportFailure([]string{err.Error()})
+			if err != nil {
+				log.Println(err.Error())
+			}
 			return
 		}
 		log.Infoln("IAM check successful")
@@ -135,7 +147,10 @@ func main() {
 		return
 	}
 
-	checkclient.ReportSuccess()
+	err = checkclient.ReportSuccess()
+	if err != nil {
+		log.Println(err.Error())
+	}
 
 }
 

